jwql/website/apps/jwql/api_views.py

```diff
@@ -206,6 +207,28 @@ def preview_images_by_rootname(request, rootname):
     return JsonResponse({'preview_images': preview_images}, json_dumps_params={'indent': 2})
 
 
+def thumbnails_all_instruments(request, inst_list):
+    """Return a list of available thumbnails in the filesystem for the
+    given request
+
+    Parameters
+    ----------
+    request : HttpRequest object
+        Incoming request from the webpage
+    inst : list of strings
```

Comment:
Looks like this should be `inst_list` instead of `inst`.  And could you change this to something like `instrument_list` (as it is a bit more explicit)?

Reply:
This function is not currently being used, so I removed it (at least temporarily), but I will make those changes if it is added back in!